src/utils/getVisibleTodos.ts

```diff
@@ -0,0 +1,17 @@
+import { Todo } from '../types/Todo';
+import { Filter } from '../types/filter';
+
+export const getVisibleTodos = (todos: Todo[], filterBy: Filter): Todo[] => {
+  return todos.filter(todo => {
+    switch (filterBy) {
+      case Filter.ALL:
+        return todo;
+      case Filter.COMPLETED:
+        return todo.completed;
+      case Filter.ACTIVE:
+        return !todo.completed;
+      default:
```

Comment:
you will never have unexpected result with typescript, move `All` to default case